demos/common/python/openvino/model_zoo/model_api/adapters/ovms_adapter.md

```diff
@@ -0,0 +1,71 @@
+# OpenVINO Model Server Adapter
+
+The `OVMSAdapter` implements `ModelAdapter` interface which makes it possible to use Model API with models hosted in OpenVINO Model Server.
+
+## Prerequisites
+
+`OVMSAdapter` enables inference via gRPC calls to OpenVINO Model Server, so in order to use it you need two things:
+- OpenVINO Model Server that serves your model
+- [`ovmsclient`](https://pypi.org/project/ovmsclient/) package installed to enable communication with the model server
+
+### Deploy OpenVINO Model Server
+
+Model Server is distributed as a docker image, so you can pull it from Docker Hub with:
+
+```
+docker pull openvino/model_server
+```
+
+See [model server documentation](https://github.com/openvinotoolkit/model_server/blob/main/docs/docker_container.md) to learn how to deploy OpenVINO optimized models with OpenVINO Model Server.
+
+### Install ovmsclient
+
+`ovmsclient` package is distributed on PyPi, so the easiest way to install it is via:
+
+```
+pip install ovmsclient
+```
+
+## Model configuration
+
+When using OpenVINO Model Server model cannot be directly accessed from the client application (like OMZ demos). Therefore any configuration must be done on model server side.
+
+### Input reshaping
+
+For some use cases you may want your model to reshape to match input of certain size. In that case, you should provide `--shape auto` parameter to model server startup command. With that option, model server will reshape model input on demand to match the input data.
+
+### Inference options
+
+It's possible to configure inference related options for the model in OpenVINO Model Server with options:
+- `--target_device` - name of the device to load the model to
+- `--nireq` - number of InferRequests
+- `--plugin_config` - configuration of the device plugin
+
+See [model server configuration parameters](https://github.com/openvinotoolkit/model_server/blob/main/docs/docker_container.md#configuration-parameters) for more details.
+
+### Example OVMS startup command
+```
+docker run -d --rm -v <models_repository>:/models -p 9000:9000 openvino/model_server:latest \
```

Comment:
Since I'm already passing some parameters with values I decided that I'll do the same with the rest of them, so now there are no `<placeholder>` type of values. Are you okay with that?